hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/processors/StreamToStreamJoinPInnerTest.java

```diff
@@ -439,8 +457,15 @@ static Expression<Boolean> createConditionFromPostponeTimeMap(
 
     private SupplierEx<Processor> createProcessor(int leftColumnCount, int rightColumnCount, int... wmKeyToColumnIndex) {
         Expression<Boolean> condition = createConditionFromPostponeTimeMap(postponeTimeMap, wmKeyToColumnIndex);
-        JetJoinInfo joinInfo = new JetJoinInfo(INNER, new int[0], new int[0], condition, condition);
+        int[] left = new int[0];
+        int[] right = new int[0];
+        if (condition instanceof ComparisonPredicate) {
+            left = new int[]{0};
```

Comment:
Looks like a leftover, `new int[]{0}` is the same as `new int[0]`.

The same problem is in `StreamToStreamJoinPOuterTest`.

Reply:
I take back, it's not the same.